agenta-web/cypress/e2e/app-navigation.cy.ts

```diff
@@ -1,14 +1,21 @@
 import {isDemo} from "../support/commands/utils"
 
 describe("App Navigation without errors", () => {
+    let app_id
+    before(() => {
+        cy.createVariant()
+        cy.get("@app_id").then((appId) => {
+            app_id = appId
+        })
+    })
+
     context("When the user navigates outside of the App views", () => {
```

Comment:
@bekossy 
this should be "..inside an app views" no ?

Reply:
Resolved